nilearn/reporting/data/html/report_body_template_niftimapsmasker.html

```diff
@@ -8,20 +8,27 @@
     <div class="image">
       <!-- prettier-ignore -->
       <script>
-        {{js_content}}
```

Comment:
OK maybe this is the reason for the js not being injected ?

Reply:
Yeah could be. From what I can remember, tempita insertions are sensitive to spaces.

Reply:
yup that was the root cause